risc0/zkp/src/hal/cpu.rs

```diff
@@ -519,6 +523,12 @@ impl<F: Field> Hal for CpuHal<F> {
             });
     }
 
+    fn eltwise_zeroize_elem(&self, elems: &Self::Buffer<Self::Elem>) {
+        elems.as_slice_mut().par_iter_mut().for_each(|elem| {
+            *elem = elem.valid_or_zero();
+        });
+    }
```

Comment:
I see that this function isn't new. It is an off name though. At first I assumed it would set the elements to zero, but it only does so if the element is `INVALID`

Reply:
Any better name?